locales/de.json

```diff
@@ -0,0 +1,144 @@
+{
+	"$name": "Deutsch (German)",
```

Comment:
You can add `de: "German"` to the english translation (and others) to handle the "(German)" part automatically depending on the currently selected language

Reply:
Ah, I see! I wonder how that worked.